tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -809,3 +809,43 @@ setup_file() {
     rm -rf $FLUVIO_HOME_DIR
     assert_success
 }
+
+@test "Disallows switching versions when there are active proccesses" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    # Starts Fluvio Cluster
+    fluvio cluster delete --local || true
```

Comment:
1. `fluvio cluster delete --local` is no longer valid in the latest.
2. Why do you stop the cluster in the test if you didn't start it? It might be started on CI earlier, and other tests might expect it. This is not the case here, but, in general, we should make tests isolated and order-independent. 
3. If the fluvio cluster is running at that place, need to investigate and find why the cluster is running in that place. Recently, I fixed similar magical cluster runs in this commit https://github.com/infinyon/fluvio/commit/ad005a5e53a647a1ce3d3e59ac98f94b7a26bb73. Maybe there is another place.

Reply:
> Why do you stop the cluster in the test if you didn't start it? It might be started on CI earlier, and other tests might expect it. This is not the case here, but, in general, we should make tests isolated and order-independent.

Yeah I thought no cluster was present, but turns out that the process detection for `fluvio` and/or `fluvio-run` determined there was one of those two processes.

> If the fluvio cluster is running at that place, need to investigate and find why the cluster is running in that place. Recently, I fixed similar magical cluster runs in this commit https://github.com/infinyon/fluvio/commit/ad005a5e53a647a1ce3d3e59ac98f94b7a26bb73. Maybe there is another place.

Agree! I also saw your PR: https://github.com/infinyon/fluvio/pull/3720 and I thought there wasn't any clusters on FVM test because of this. But perhaps I still have to dig in on this.